index.js

```diff
@@ -215,13 +221,25 @@ exports.viewAccount = async function (options) {
 
 exports.deleteAccount = async function (options) {
     await checkCredentials(options.accountId, options.networkId, options.keyStore);
-    console.log(
-        `Deleting account. Account id: ${options.accountId}, node: ${options.nodeUrl}, helper: ${options.helperUrl}, beneficiary: ${options.beneficiaryId}`);
     const near = await connect(options);
-    const account = await near.account(options.accountId);
-    const result = await account.deleteAccount(options.beneficiaryId);
-    inspectResponse.prettyPrintResponse(result, options);
-    console.log(`Account ${options.accountId} for network "${options.networkId}" was deleted.`);
+    const beneficiaryAccount = await near.account(options.beneficiaryId);
+    // beneficiary account does not exist if there are no access keys
+    if (!(await beneficiaryAccount.getAccessKeys()).length) {
```

Comment:
@judith-Near If the account has 0 AC. itdoes not mean it does not exist.

Reply:
+ geAccessKeys() can return `null`, it will crash the execution.

Reply:
Ah... I misunderstood what's happening here. I was under the impression that this would throw an error if the account doesn't exist and you're trying to query for the keys. That's on me. Apologies.

Reply:
Hm, maybe it will. It's better to test it. But logic seems wrong anyway.

Reply:
Addressed here: https://github.com/near/near-cli/pull/1009/files